inspirehep/modules/workflows/views/callback.py

```diff
@@ -97,7 +148,13 @@ def webcoll_callback():
 
     Expects the request data to contain a list of record ids in the
     recids field.
+
+    DEPRECATED: replaced by the simpler continue_workflow_callback
```

Comment:
@david-caro Ideally it could sit in a new SQL table, if you want to take the pain to port it to it :+1: I guess redis had been chosen because this was anyway a quick hack.